uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -174,6 +175,7 @@ define([
 
     describe('isRole()', function () {
       beforeEach(function () {
+
```

Comment:
I think you need to remove that empty line and put an empty line after the initController(); because the next line is assignment.